xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -390,8 +390,6 @@ namespace XBMCAddon
             item->GetVideoInfoTag()->m_strProductionCode = value;
           else if (key == "aired")
             item->GetVideoInfoTag()->m_firstAired.SetFromDateString(value);
-          else if (key == "credits")
-            item->GetVideoInfoTag()->m_writingCredits = StringUtils::Split(value, g_advancedSettings.m_videoItemSeparator);
```

Comment:
Well, then I put it in other words, it doesnt need script authors to adjust their scripts in order to make things work. A "real" API breaker is this here for example: https://github.com/xbmc/xbmc/pull/9187

I doubt anyone even knew what setInfo("credits") even did, and the worst fallout there can be is a missing ListItem.Writer InfoLabel for plugin listitems...
A git grep through add-on repo by @ronie would be cool to see if it is used at all. I doubt it.